bitswap/client/internal/session/sessionwantsender.go

```diff
@@ -577,11 +570,6 @@ func (sws *sessionWantSender) sendNextWants(newlyAvailable []peer.ID) {
 func (sws *sessionWantSender) sendWants(sends allWants) {
 	// For each peer we're sending a request to
 	for p, snd := range sends {
-		// Piggyback some other want-haves onto the request to the peer
-		for _, c := range sws.getPiggybackWantHaves(p, snd.wantBlocks) {
-			snd.wantHaves.Add(c)
-		}
-
```

Comment:
Are we sure removing this has no effect?

swbt was introduced here: https://github.com/ipfs/go-bitswap/commit/b3a47bcf5080c734346fc39400802c35aedd6428#diff-92a0b168d045013a05e87675741bb3cfa356a674c6f1535fbe88fc8e89d31e53

It is not explained what exactly motivated it, but it sounds like it is somehow keeping track of some wants that would otherwise not be sent?